status-im/status-react
# [#7927] import keycard account to new device

fixes #7913

### Summary

Add support for importing keycard account to new device.

### Steps to test

1. Set up keycard on device A
2. Add existing account to device B
3. Enter pairing code
4. Start importing account
5. Account imported, you should be able to log in to device B
6. You should be also able to log in to device A

status: ready
